### ISSUE TITLE
lack of tagline / context on homepage

### ISSUE THREAD

**Issue by evanwill** [uidaholib/inside_source#25]
@b-godfrey looking at the inside home page, I wonder if there should be a little intro tagline giving context somewhere near the top of the home page. 
I just find scanning down the page, I see the nav, then a search bar, a carousel with data items, then "university of idaho resources", but I still don't exactly know what "inside" is or a context of where I am?

**Comment by b-godfrey** [comment]
Sure, I'm open to adding that.  I believe the old site had something like "Idaho's Geospatial Data Clearinghouse".  I could look it up.  I'm open to any variation though -- "GIS data for the State of Idaho", etc.  suggestions?

________________________________

**Comment by evanwill** [comment]
Yeah, I am not sure-- no rush to sort out, I just wanted to make a note while I was thinking about it so I didn't forget to bring up.
